packages/core/src/app/payment/Payment.tsx

```diff
@@ -602,11 +603,13 @@ export function mapToPaymentProps({
     const config = getConfig();
     const customer = getCustomer();
     const consignments = getConsignments();
+    const paymentProviderCustomer = getPaymentProviderCustomer();
+
     const { isComplete = false } = getOrder() || {};
     let methods = getPaymentMethods() || EMPTY_ARRAY;
 
     // TODO: In accordance with the checkout team, this functionality is temporary and will be implemented in the backend instead.
-    if (customer?.isStripeLinkAuthenticated) {
+    if (paymentProviderCustomer?.stripeLinkAuthenticationState) {
```

Comment:
This line might throw an error after you pull last changes from master

Reply:
It is working fine